data/org.freedesktop.portal.Device.xml

```diff
@@ -58,6 +58,66 @@
       <arg type="a{sv}" name="options" direction="in"/>
       <arg type="o" name="handle" direction="out"/>
     </method>
+
+    <!--
+        OpenPipeWireRemote:
+        @options: Vardict with optional further information
+        @fd: File descriptor of an open PipeWire remote.
+
+        Open a file descriptor to the PipeWire remote where the microphone,
+        speakers, and camera nodes are available. The file descriptor should be
+        used to create a <classname>pw_core</classname> object, by using
+        <function>pw_context_connect_fd</function>.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>devices as</term>
+            <listitem><para>
+              The devices to access. Supported devices for this call are 'camera',
+              'microphone', and 'speakers'.
```

Comment:
I was poking around the OBS Studio PRs regarding things like virtual cameras, and I wonder if adding the ability for the portal to ask the user to let an app create virtual webcams would be a good idea?